### PR TITLE
Checkboxes update their color

### LINKED ISSUE #70: Check item color doesn't update
When changing the task list color, check items should update to match the new color

### PR DESCRIPTION
**Pull request by marbetschar**
Fixes #70:

Checkbox Color

### CODE REVIEW

File: data/TaskRow.css [hunk 1]
```diff
@@ -48,6 +48,11 @@ row.card {
     padding-top: 6px;
 }
 
+row check {
+    padding: 1px;
+    border-radius: 12px;
+}
```
Comment (danrabbit) [inline data/TaskRow.css:54]: I'd prefer not to do this since we use the rounded shape elsewhere to indicate radio buttons

Comment (marbetschar) [reply data/TaskRow.css]: Personally I think it looks much nicer, especially together with the potential spinner of #104 - but don't have too strong feelings about this. Removed the css style.

File: src/Widgets/TaskRow.vala [hunk 2]
```diff
@@ -75,7 +75,7 @@ public class Tasks.TaskRow : Gtk.ListBoxRow {
 
         check = new Gtk.CheckButton ();
         check.valign = Gtk.Align.CENTER;
-        Tasks.Application.set_task_color (source, check);
+        check.get_style_context ().add_provider (taskrow_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
```
Comment (danrabbit) [inline src/Widgets/TaskRow.vala:78]: Is this necessary now?

Comment (marbetschar) [reply src/Widgets/TaskRow.vala]: @danrabbit oh boy, seems like it was already late... Good catch! Of course this is no longer needed. Removed.

**Pull request merged**